app/services/vendor_status.rb

```diff
@@ -90,7 +90,7 @@ def track_event(analytics)
       },
       redirect_from: from,
     }
-    analytics.track_event(Analytics::VENDOR_OUTAGE, tracking_data)
+    analytics.vendor_outage(**tracking_data)
```

Comment:
same as before, let's inline the `tracking_data` variable?
```suggestion
    analytics.vendor_outage(
      vendor_status: {
        # ...
      },
      redirect_from: from,
    )
```